### ISSUE TITLE
[Public Beta] Issue Hierarchy powered by Tasklists

### ISSUE THREAD

**Issue by github-product-roadmap** [github/roadmap#760]
### Summary

![image](https://user-images.githubusercontent.com/101840513/252498143-53087b43-f2f2-45af-8d9d-f7946f579231.png)

Users will now have the ability to navigate, explore and work with a hierarchy of Issues on GitHub!

## Feature Highlights: 
* Tasklists are powered by a unique Markdown formatting and can easily be added to Issue Templates or created via the API
* Create multi-level hierarchies of Issues from the Issues page or GitHub Projects
* See relevant labels, assignees, and child issues in tasklists
* Explore the hierarchy using breadcrumbs in the Projects side-panel
* Keep track of Issue tasklist completion with the new "Tracks" and "Tracked by" columns in Projects 
* Quickly edit Issue meta-data with in-line editing capabilities 

### Intended Outcome

Managing software projects at scale can be challenging without Issue hierarchies. Customers want the ability to create large batches of work (sometimes called Epics) and break those batches down into smaller more manageable tasks; all while tracking progress in a single GitHub Issue or from GitHub Projects.

### How will it work?

See docs: https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists

In short, custom Markdown formatting with custom rendering, will allow users to create structured Hierarchies of Issues. 

---
If you'd like to gain early access, sign up for the Private Beta here: https://github.com/features/issues/signup

**Comment by ankneis** [comment]
Thank you to everyone who has participated in this beta and helped provide feedback on tasklists. Tasklists will be evolving into sub-issues, more details of this can be [found here](https://github.com/github/roadmap/issues/927.)

We will be providing a migration from tasklists to sub-issues and will provide more details in the future.

**Comment by ankneis** [comment]
We wanted to provide more details on why we removed this from the roadmap. We have shipped an alternative implementation of this with the recent release of [sub-issues](https://github.blog/changelog/2024-10-01-evolving-github-issues-public-beta/).

Check out our dedicated [sub-issues feedback discussion](https://github.com/orgs/community/discussions/139932) and share your thoughts and help us shape this feature.
